e2e/validate/storageclass.go

```diff
@@ -0,0 +1,48 @@
+package validate
+
+import (
+	"context"
+	"fmt"
+
+	storagev1 "k8s.io/api/storage/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/ramendr/ramen/e2e/types"
+)
+
+// validateStorageClasses verifies that each storageClass referenced in PVC specs
+// exists in both managed clusters.
+func validateStorageClasses(env *types.Env, config *types.Config) error {
+	clusters := []*types.Cluster{&env.C1, &env.C2}
+
+	for _, spec := range config.PVCSpecs {
+		for _, cluster := range clusters {
+			_, err := getStorageClass(cluster, spec.StorageClassName)
+			if err != nil {
+				return fmt.Errorf("failed to find storageClass %q in cluster %q for pvcSpec %q: %w",
+					spec.StorageClassName, cluster.Name, spec.Name, err)
```

Comment:
The original error already include the cluster and storageclass name, so we don't want to wrapp it again here with the same values. We can use:

    return fmt.Errorf("failed to pvcSpec %q storage class: %w", spec.Name, err)

Please share the error for missing storage class in the commit message to make the change more clear and easier to review.